textdata_reader.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
```

Comment:
Let's change copyright owner to Petuum insted of The Forte Authors.

Also time is 2021